config/locales/fr.yml

```diff
@@ -125,12 +126,22 @@ fr:
         wiki: "Wiki"
         mail: "Liste de diffusion"
         bugs: "Suivi de bugs"
+        download: "Télécharger"
+        subscribe: "Abonnement"
```

Comment:
Yeah, I guess. I just hate this style of semi-condescending verb-based actions in French. Adverbs always feel more proper and less cutesy.